kokoro/common.cfg

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
all the kokoro cfg and script file changes should be removed. Master has the latest config.